app/lib/activitypub/activity/create.rb

```diff
@@ -336,13 +336,15 @@ def resolve_thread(status)
 
   def fetch_replies(status)
     collection = @object['replies']
-    return if collection.nil?
+    return if collection.blank?
 
     replies = ActivityPub::FetchRepliesService.new.call(status, collection, allow_synchronous_requests: false, request_id: @options[:request_id])
     return unless replies.nil?
 
     uri = value_or_id(collection)
     ActivityPub::FetchRepliesWorker.perform_async(status.id, uri, { 'request_id' => @options[:request_id] }) unless uri.nil?
+  rescue e
```

Comment:
It think this should be `rescue => e`, should it not? And do you really want to rescue all (standard)errors here? (It might make sense, and I might propose this in another context as well, just want to make sure.)

Reply:
Oops, good catch. Yes, catching all standard errors was the intent, in this pretty part that is pretty much non-essential and highly reliant on remote data. We could avoid that, but I'm not confident we won't hit other malformed activities that would trigger other such bugs.